### PR TITLE
initial recipe for mixsimulator

### PR DESCRIPTION
**Pull request by JennaLipscomb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JennaLipscomb** [comment]
agree to maintain @oblute

**Comment by JennaLipscomb** [comment]
@conda-forge/staged-recipes ready for review

**Comment by oblute** [comment]
@conda-forge/staged-recipes

File: recipes/mixsimulator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - mixsimulator
+    - mixsimulator.centrals
+
+about:
+  home: https://github.com/Foloso/MixSimulator
+  summary: Python application with nevergrad optimization model for calculating and simulating the least cost of an energy Mix under constraints.
+  license_file: LICENSE.txt
+  license: MIT
+  license_family: MIT
+  doc_url: https://facebookresearch.github.io/nevergrad/
```
Comment (synapticarbors) [inline recipes/mixsimulator/meta.yaml:36]: This `doc_url` and `dev_url` look to be incorrect

File: recipes/mixsimulator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - mixsimulator
+    - mixsimulator.centrals
+
+about:
+  home: https://github.com/Foloso/MixSimulator
+  summary: Python application with nevergrad optimization model for calculating and simulating the least cost of an energy Mix under constraints.
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/mixsimulator/meta.yaml:33]: I opened a PR upstream to deal with having to manually package the license file (https://github.com/Foloso/MixSimulator/pull/5). 

Please check this upon the next release to see if you can drop manually packaging the license file.

**Review by synapticarbors — COMMENTED** [review]
@ocefpaf merged before I finished this review. Please note the following comments.

File: recipes/mixsimulator/meta.yaml (hunk 1 above)
Comment (JennaLipscomb) [reply recipes/mixsimulator/meta.yaml:36]: You're correct! Must have been a copy/paste error, the correct urls should be: https://github.com/Foloso/MixSimulator 

Should I do an update PR?

Comment (synapticarbors) [reply recipes/mixsimulator/meta.yaml:36]: You'll need to open a PR in your person fork of the feedstock (https://github.com/conda-forge/mixsimulator-feedstock). You can just increment the build number as well and rerender the feedstock to update the README and other metadata. Let me know if you need any assistance or to take a look at the PR.

**Pull request merged**